vendor/src/github.com/docker/engine-api/types/registry/registry.go

```diff
@@ -82,7 +82,7 @@ type SearchResult struct {
 	IsOfficial bool `json:"is_official"`
 	// Name is the name of the repository
 	Name string `json:"name"`
-	// IsOfficial indicates whether the result is trusted
+	// IsTrusted indicates whether the result is trusted
```

Comment:
Hum, it's already in engine-api, and I guess it's for `golint`, is the attribute is named `IsTrusted` :angel: